internal/controllers/import_controller.go

```diff
@@ -271,7 +247,7 @@ func (r *CAPIImportReconciler) reconcileNormal(ctx context.Context, capiCluster
 	}
 
 	// get the registration manifest
-	manifest, err := r.getClusterRegistrationManifest(ctx, rancherCluster.Status.ClusterName, capiCluster.Namespace)
+	manifest, err := getClusterRegistrationManifest(ctx, rancherCluster.Status.ClusterName, capiCluster.Namespace, r.Client, r.InsecureSkipVerify)
```

Comment:
Here the r.RancherClient should be used, see https://github.com/rancher-sandbox/rancher-turtles/pull/362/files#diff-df81a5620bfede50d1c70fd0ebb982e652d28721e15b825c52dddcf97ceda0deL337

Reply:
Thanks for noticing. These two are unintended errors because of naming similarities. Fixing them now.